ops/prod/persistent/main.tf

```diff
@@ -70,4 +70,4 @@ module "vnet" {
   network_address     = local.network_cidr
   management_tags     = local.management_tags
   location            = local.rg_location
-}
+}
```

Comment:
what changed here?

Reply:
The newline at the end of the file was removed.